userspace/libsinsp/settings.h

```diff
@@ -24,6 +24,19 @@ limitations under the License.
 //
 #define SP_EVT_BUF_SIZE 4096
 
+//
+// Controls if assertions break execution or if they are just printed to the
+// log. It's not defined here, because some asserts are used even before
+// settings header is included, so it's easier to provide it at the compile
+// time.
+//
+// #define ASSERT_TO_LOG
+
+//
+// Controls if the library collects internal performance stats.
+//
+#undef GATHER_INTERNAL_STATS
```

Comment:
Didn't know we disabled it ourselves. Was there any particular reason for that?

Reply:
Not sure; I think it was part of the log asserts changes